# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1213947295 -7200
#      Fri Jun 20 09:34:55 2008 +0200
# Branch release-3-0-x
# Node ID 9de3ccd2e7acd52d69ce239876ef3dd4f5f197d1
# Parent  35d5c824f1eb3b35ff08cee9419e058bb84fed0a
add missing include in MatrixType.h

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2008-06-20  Jaroslav Hajek <highegg@gmail.com>
+
+	* MatrixType.h: Add missing include statement.
+
 2008-05-19  David Bateman  <dbateman@free.fr>
 
 	* dSparse.cc: Replace some DGBCON with GPBCON where they are
 	incorrectly used.
 
 	* idx-vector.cc (IDX_VEC_REP::idx_vector_rep (const boolNDArray&)):
 	If len is zero size the index vector in the same manner as if len
 	is not zero.
diff --git a/liboctave/MatrixType.h b/liboctave/MatrixType.h
--- a/liboctave/MatrixType.h
+++ b/liboctave/MatrixType.h
@@ -19,16 +19,18 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if !defined (octave_MatrixType_h)
 #define octave_MatrixType_h
 
+#include "oct-types.h"
+
 class Matrix;
 class ComplexMatrix;
 class SparseMatrix;
 class SparseComplexMatrix;
 
 class
 OCTAVE_API
 MatrixType
